chart/templates/deployment.yaml

```
@@ -123,6 +127,9 @@ spec:
             {{- toYaml .Values.exporter.resources | nindent 12 }}
           {{- end }}
         {{- end }}
+        {{- if .Values.extraContainers }}
+          {{- toYaml .Values.extraContainers | nindent 10 }}
```

Is the indentation correct? If `.Values.extraContainers` is a list, this should be indented by 8 spaces if I'm not mistaken.

```suggestion
          {{- toYaml .Values.extraContainers | nindent 8 }}
```

EDIT, which would incidentally be the same indentation level as already used in `statefulset.yaml`.